### ISSUE TITLE
GetPlayerByID throws exception when playerID is not in players Dictionary

### ISSUE THREAD

**Issue by Guribo** [CyanLaser/CyanEmu#31]
When trying to get a player by an invalid ID e.g. -1 it throws an exception instead of returning null.

Issue: 
![image](https://user-images.githubusercontent.com/49484893/105395117-af9cdf80-5c1e-11eb-9b81-3a4384073d40.png)  

https://github.com/CyanLaser/CyanEmu/blob/6b959f5a109e89c89963137dfb6ba73e78688728/CyanEmu/Scripts/CyanEmuPlayerManager.cs#L112

Solution:
```
        public static VRCPlayerApi GetPlayerByID(int playerID)
        {
            return players.TryGetValue(playerID, out var player) ? player : null;
        }
```

**Comment by CyanLaser** [comment]
Thank you for reporting this. I have implemented a fix in da7b43174621ac4bcd064186c7bf217f2c5fbea3.

**Comment by Guribo** [comment]
Thanks!
